tbtadm/controller.cpp

```diff
@@ -746,12 +746,17 @@ void tbtadm::Controller::acl()
 
 void tbtadm::Controller::add(const fs::path& dir)
 {
-    if (m_sl == 2)
+    switch (m_sl)
     {
-        m_out << "Adding to ACL on SL2 must be done together with device "
-                 "approval\n";
-        return;
+        case 2:
+            m_out << "Adding to ACL on SL2 must be done together with device "
+                    "approval\n";
+            return;
+        case 0:
```

Comment:
OK, if we want this, it should be similar to line 560 above (or even refactored into a function called from both locations). SL3 is also a thing.

Reply:
I think we need to preserve error messages which are different for every security level. Or do you mean we need to refactor also line 560 the same way? I can also include `default` case for other levels.

Reply:
This must capture only SL1. Any other SL isn't supported here, each one for its own reason.